### DISCUSSION TITLE
Terminal profile 'run this profile as administrator'

### DISCUSSION THREAD

**Discussion by blakeduffey-noblis** [microsoft/terminal#15691] [General]
User is a non-admin (corporate environment)
Terminal was installed/updated via winget 
I create a profile with 'run this profile as administrator' enabled (in this case, just launching powershell)

I launch that profile - get prompted to authenticate as an administrator (which I expect) - then I get an error:

This app can't open
The application cannot be started for the target user.

**Comment by rlwakefield** [comment]
I just tested this on release 1.19.3172.0 and it worked fine for me (I am in a corporate environment as well). So I wonder if it was fixed later on.

NOTE: I know there were issues when running Terminal as Admin for a while. So I wonder if when that was fixed, this might have been fixed as well.

**Comment by blakeduffey** [comment]
@rlwakefield  - confirmed here as well.  Also on 1.19.3172.9 - very useful feature!

![image](https://github.com/microsoft/terminal/assets/5239758/3523a5ae-5234-476e-b922-440b8fed32f6)

**Comment by rlwakefield** [comment]
Agreed @blakeduffey. Now we just need to make it so that when you launch Windows Terminal in general, it launches as administrator. Right now you set it at a per profile basis, but it would be fantastic to be able to have it so that when you launch WT that it loads as administration. I have tried setting the executable and everything to run as admin, but nothing seems to work. 😞
